pkg/gardener/shoot/hyperscaler/alicloud/config.go

```diff
@@ -0,0 +1,51 @@
+package alicloud
+
+import (
+	"encoding/json"
+
+	"github.com/gardener/gardener-extension-provider-alicloud/pkg/apis/alicloud/v1alpha1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	infrastructureConfigKind = "InfrastructureConfig"
+	controlPlaneConfigKind   = "ControlPlaneConfig"
+	apiVersion               = "alicloud.provider.extensions.gardener.cloud/v1alpha1"
+)
+
+func GetInfrastructureConfig(workerCIDR string, zones []string) ([]byte, error) {
+	return json.Marshal(NewInfrastructureConfig(workerCIDR, zones))
```

Comment:
Suggestion- return nil if NewInfrastructureConfig returns an error - like for AWS and Azure case